lib/flutter_zalo_login.dart

```diff
@@ -43,7 +43,8 @@ class ZaloLogin {
 
   /// [isAuthenticated] will check is session out or not
   /// Ex: `bool isAuthenticated = await ZaloLogin().isAuthenticated();`
-  Future<bool> isAuthenticated() async => await channel.invokeMethod('isAuthenticated') == 1;
+  Future<bool> isAuthenticated() async =>
+      await channel.invokeMethod('isAuthenticated') == 1;
```

Comment:
**Suggestion:** Remove unnecessary await in the return statement of isAuthenticated method.
```suggestion
  Future<bool> isAuthenticated() async =>
      channel.invokeMethod('isAuthenticated') == 1;
```